src/pretix/static/pretixpresale/js/ui/cookieconsent.js

```diff
@@ -0,0 +1,103 @@
+/*global $ */
+
+$(function () {
+    function _send_event(type) {
+        // Event() is not supported by IE11
+        var e = document.createEvent('Event')
+        e.initEvent(type, true, true)
+        document.dispatchEvent(e)
+    }
+
+    if ($("#cookie-consent-storage-key").length) {
+        window.__pretix_cookie_consent = {}
+    } else {
+        window.__pretix_cookie_consent = null
+        return
+    }
+
+    var storage_key = $("#cookie-consent-storage-key").text()
+    var storage_val = window.localStorage[storage_key]
+    var show_dialog = false
+    if (!storage_val) {
+        show_dialog = true
+        storage_val = {}
+
+        if ($("#cookie-consent-from-widget").length) {
+            var consented = JSON.parse($("#cookie-consent-from-widget").text())
+            $("#cookie-consent-details input[type=checkbox][name]").each(function () {
+                storage_val[$(this).attr("name")] = consented.indexOf($(this).attr("name")) > -1
```

Comment:
No need to jquery here – also in a few other places.

```suggestion
                storage_val[this.name] = consented.indexOf(this.name) > -1
```